src/Plugin.php

```diff
@@ -108,6 +108,15 @@ public static function init() {
     // Adds product attributes to order emails.
     add_filter('woocommerce_display_item_meta', __NAMESPACE__ . '\WooCommerce::woocommerce_display_item_meta', 10, 3);
 
+    if (class_exists('Woocommerce_German_Market')) {
```

Comment:
1. The condition seems unnecessary/unwanted, as the variable `show_sku` is also used by WooCommerce itself:
https://github.com/netzstrategen/gaco/blob/858888251d7f559c19be82b5bbf4a0cdd7548279/wp-content/plugins/woocommerce/templates/emails/plain/email-order-details.php#L27

1. I would recommend to move the newly added lines directly below/above the (second to last) filter hook `woocommerce_get_item_data` (line 106) above, so that customizations on the same aspect are grouped together.

Reply:
> The condition seems unnecessary/unwanted, as the variable show_sku is also used by WooCommerce itself...

@sun It seems to me that WooCommerce is using this to add the SKU to the top-over-the-header (before the order table) of notification emails. We only want to remove SKU from item names in the order items table, which is added there by WGM.

> I would recommend to move the newly added lines directly below/above the (second to last) filter hook woocommerce_get_item_data (line 106) above, so that customizations on the same aspect are grouped together.

I will do so.

Reply:
That's the same spot as WGM; both are targeting the list of order items. WGM is overriding the WooCommerce template. WooCommerce outputs the SKU by default. WGM adds the delivery time to the output.

Reply:
@sun My bad. Closed the PR before seeing your last comment 🙈 You are totally right here. I will add those fixes and create a new PR.